.github/workflows/feature_6_docs.yml

```diff
@@ -0,0 +1,63 @@
+name: Develop > Tiny docs 6.0
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - '*'
+    branches:
+      - 'feature-6/**'
+
+jobs:
+  build:
+    name: Build Docs and Deploy
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [14]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        cache: 'yarn'
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Antora@3.0.1
+      run: sudo yarn global add @antora/cli@3.0.1
+
+    - name: (Feature - Docs 6.0) Install dependencies
+      run: yarn install -g
```

Comment:
Why are we doing this? We have a package.json and lockfile for a reason... we should just be able to run `yarn install` instead here.

Reply:
My bad, I didn't clean it well